app/models/custom_filter.rb

```diff
@@ -24,6 +24,29 @@ class CustomFilter < ApplicationRecord
   enum filter_type: { conversation: 0, contact: 1, report: 2 }
   validate :validate_number_of_filters
 
+  def records_count
+    fetch_record_count_from_redis
+  end
+
+  def filter_records
+    user = account.users.try(:first)
+    Conversations::FilterService.new(query.with_indifferent_access, user, account).perform
+  end
+
+  def set_record_count_in_redis
```

Comment:
Is updating the filter possible? We should invalidate the record count if the filter is updated, we can be a bit smart about it by ensuring that the filter conditions have changed or not.

Reply:
Yes, we have a ticket to update it on the conversation update. https://linear.app/chatwoot/issue/CW-1611/refresh-the-count-without-reload